PennMobile/src/main/res/values/colors.xml

```diff
@@ -10,6 +10,7 @@
     <color name="olive">#808000</color>
     <color name="purple">#800080</color>
     <color name="maroon">#800000</color>
+    <color name ="PennRed">#990000</color>
```

Comment:
Why do you have this just to not use it?

Reply:
It would be used spectacularly in the future (Baron is coping)